docs-sphinx/Makefile

```diff
@@ -1,70 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
When reorganising folder structure, use git mv if not using an IDE with git integration.
This avoids the review bloating where it looks like you deleted the contents of files and created files.

Reply:
I did that with a new branch as I am doing everything all over again.